packages/electron/lib/install.js

```diff
@@ -25,44 +23,66 @@ module.exports = {
   // returns icons package so that the caller code can find
   // paths to the icons without hard-coding them
   icons () {
-    return require('@cypress/icons')
+    return require('@packages/icons')
   },
 
   checkCurrentVersion () {
     const pathToVersion = paths.getPathToVersion()
 
     // read in the version file
-    return fs.readFileAsync(pathToVersion, 'utf8')
+    return fs.readFile(pathToVersion, 'utf8')
     .then((str) => {
       const version = str.replace('v', '')
 
       // and if it doesn't match the electron version
       // throw an error
       if (version !== electronVersion) {
         throw new Error(`Currently installed version: '${version}' does not match electronVersion: '${electronVersion}`)
-      } else {
-        return process.exit()
+      }
+    })
+  },
+
+  getFileHash (filePath) {
+    return fs.readFile(filePath).then((buf) => {
+      const hashSum = crypto.createHash('sha1')
+
+      hashSum.update(buf)
+      const hash = hashSum.digest('hex')
+
+      return hash
+    })
+  },
+
+  checkIconVersion () {
+    const mainIconsPath = this.icons().getPathToIcon('cypress.icns')
+    const cachedIconsPath = path.join(__dirname, '../dist/Cypress/Cypress.app/Contents/Resources/electron.icns')
+
+    return Promise.all([this.getFileHash(mainIconsPath), this.getFileHash(cachedIconsPath)])
+    .then(([mainHash, cachedHash]) => {
+      console.log({ mainHash, cachedHash })
```

Comment:
do we need this console log?

Reply:
no you are right good catch!
I removed it